packages/blockchain-extension/extension/commands/openTransactionViewCommand.ts

```diff
@@ -41,6 +42,9 @@ export async function openTransactionView(treeItem?: InstantiatedTreeItem): Prom
         }
     }
 
+    const gatewayRegistryEntry: FabricGatewayRegistryEntry = FabricGatewayConnectionManager.instance().getGatewayRegistryEntry();
+    const gatewayName: string = (gatewayRegistryEntry.name === 'local_fabric') ? 'Local Fabric' : gatewayRegistryEntry.name;
```

Comment:
FabricRuntimeUtil.ts has constants for these these two!

Reply:
Fixed 👍